tests/Fixtures/Issue/Issue156Contact.php

```diff
@@ -69,7 +68,7 @@ public function removeTelephoneNumber(Issue156ContactTelephoneNumber $telephoneN
     }
 
     /**
-     * @return ArrayCollection|Issue156ContactTelephoneNumber[]
+     * @return iterable<int, Issue156ContactTelephoneNumber>
```

Comment:
Dunno why iterable is used as typehint, it could be `Collection` instead isn't it ?